dev_guide/builds.adoc

```diff
@@ -1025,10 +1025,113 @@ is *dockerhub*:
 == Using Private Repositories for Builds
 
 Supply valid credentials to build an application from a private repository.
-Currently, only SSH key based authentication is supported. The repository keys
-are located in the `$HOME/.ssh/` directory, and are named `id_dsa.pub`,
-`id_ecdsa.pub`, `id_ed25519.pub`, or `id_rsa.pub` by default. Generate SSH key
-credentials with the following command:
+
+Currently two types of authentication are supported. Basic username-password
+and SSH key based authentication.
+
+[#basic-authentication]
+=== Basic Authentication
+
+Basic authentication requires either a combination of `username` and `password`,
+or a `token` to authenticate against the SCM server. A `CA certificate` file,
+or a `.gitconfig` file can be attached.
+
+A link:dev_guide/secrets[`*secret*`] is used to store your keys.
```

Comment:
Should be:

`link:../dev_guide/secrets.html`

or just:

`link:secrets.html`